docs/API.Auth.GeneratePassword.md

```diff
@@ -0,0 +1,30 @@
+# Function: *Generate Password* #
+
+## Description ##
+
+When creating a new CM-Well user, you can use the **generate-password** API to generate a random password and its basic-encoded value. This password can be changed later if required (see [Change Password API](API.Auth.ChangePassword.md)).
+
+>**Note:** You will need an access token with admin permissions to call this API.
+
+## Syntax ##
+
+**URL:** \<CMWellHost\>/_auth?op=generate-password -H "X-CM-Well-Token:\<AdminToken\>"
+**REST verb:** GET
+**Mandatory parameters:** Admin-permissions token in the **X-CM-Well-Token** header
+
+## Code Example ##
+
+### Call ###
+
+    curl <cm-well-host>/_auth?op=generate-password -H "X-CM-Well-Token:<AdminToken>"
```

Comment:
just "curl <cm-well-host>/_auth?op=generate-password".

@nruppin sorry if I was misleading you about this specific API. I might have said once "all _auth operations required an Admin token" which was clearly a mistake - this API is an exception.